app/assets/javascripts/discourse/tests/acceptance/sidebar-plugin-api-test.js

```diff
@@ -874,4 +874,121 @@ acceptance("Sidebar - Plugin API", function (needs) {
       resetCustomTagSectionLinkPrefixIcons();
     }
   });
+
+  test("New custom sidebar panel and option to set default", async function (assert) {
+    withPluginApi(PLUGIN_API_VERSION, (api) => {
+      api.addSidebarPanel((BaseCustomSidebarPanel) => {
+        const ChatSidebarPanel = class extends BaseCustomSidebarPanel {
+          get key() {
+            return "new-panel";
+          }
+          get switchButtonLabel() {
```

Comment:
```suggestion
          }

          get switchButtonLabel() {
```

Minor for consistency sake